service/resources/server_resource.go

```diff
@@ -181,392 +177,34 @@ func (s ServerResource) QueryHandler(w http.ResponseWriter, r *http.Request) {
-					util.Ports:           d.Spec.Ports,
-					util.Cluster:         clusterName,
-					util.ResourceVersion: d.ObjectMeta.ResourceVersion,
-					util.K8sObj:          util.K8sObj,
-				}
-				// creata application from labels
-				appList := createAppNameList(&d)
-				if len(appList) > 0 {
-					service[util.Application] = appList
-				}
-				list = append(list, service)
-			}
-			return list, nil
-		}
-		data := core_v1.Service{}
-		err := json.Unmarshal(body, &data)
-		if err != nil {
-			return nil, err
-		}
-		service := map[string]interface{}{
-			util.ObjType:         "Service",
-			util.Name:            data.ObjectMeta.Name,
-			util.Namespace:       data.ObjectMeta.Namespace,
-			util.CreationTime:    data.ObjectMeta.CreationTimestamp,
-			util.Selector:        data.Spec.Selector,
-			util.Labels:          data.ObjectMeta.GetLabels(),
-			util.ClusterIP:       data.Spec.ClusterIP,
-			util.ServiceType:     data.Spec.Type,
-			util.Ports:           data.Spec.Ports,
-			util.Cluster:         clusterName,
-			util.ResourceVersion: data.ObjectMeta.ResourceVersion,
-			util.K8sObj:          util.K8sObj,
-		}
-		// creata application from labels
-		appList := createAppNameList(&data)
-		if len(appList) > 0 {
-			service[util.Application] = appList
-		}
-		return service, nil
-	case "StatefulSet":
-		if isArray {
-			list := make([]map[string]interface{}, 0)
-			data := []appsv1.StatefulSet{}
-			err := json.Unmarshal(body, &data)
-			if err != nil {
-				return nil, err
-			}
-			for _, d := range data {
-				statefulset := map[string]interface{}{
-					util.ObjType:         "StatefulSet",
-					util.Name:            d.ObjectMeta.Name,
-					util.CreationTime:    d.ObjectMeta.CreationTimestamp,
-					util.Namespace:       d.ObjectMeta.Namespace,
-					util.NumReplicas:     d.Spec.Replicas,
-					util.Cluster:         clusterName,
-					util.ResourceVersion: d.ObjectMeta.ResourceVersion,
-					util.Labels:          d.ObjectMeta.GetLabels(),
-					util.K8sObj:          util.K8sObj,
-				}
-				list = append(list, statefulset)
-			}
-			return list, nil
-		}
-		data := appsv1.StatefulSet{}
-		err := json.Unmarshal(body, &data)
-		if err != nil {
-			return nil, err
-		}
-		return map[string]interface{}{
-			util.ObjType:         "StatefulSet",
-			util.Name:            data.ObjectMeta.Name,
-			util.CreationTime:    data.ObjectMeta.CreationTimestamp,
-			util.Namespace:       data.ObjectMeta.Namespace,
-			util.NumReplicas:     data.Spec.Replicas,
-			util.Cluster:         clusterName,
-			util.ResourceVersion: data.ObjectMeta.ResourceVersion,
-			util.Labels:          data.ObjectMeta.GetLabels(),
-			util.K8sObj:          util.K8sObj,
-		}, nil
-	default:
-		var data interface{}
-		if isArray {
-			data = []map[string]interface{}{}
-		} else {
-			data = map[string]interface{}{}
-		}
-		err := json.Unmarshal(body, &data)
-		if err != nil {
-			return nil, err
+func (s *ServerResource) QSLHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could you pull the latest code for server_resource.go and make your change base on that? You change seems overwrite that file and some code changes will lost